orttraining/orttraining/python/training/orttrainer.py

```diff
@@ -254,6 +254,10 @@ def train_step(self, *args, **kwargs):
         input_desc = [*self.model_desc.inputs, self.model_desc.learning_rate]
         output_desc = self.model_desc.outputs
 
+        # Update Learning Rate if Necessary
+        if self.options.lr_scheduler:
+            self.options.lr_scheduler._step(self._train_step_info)
```

Comment:
This is not strictly related to this PR, but changing `LRScheduler._step` to `LRSCheduler.step` would help future extensions in which users would be able to call `step` themselves.